Libraries/StyleSheet/StyleSheet.js

```diff
@@ -59,10 +68,37 @@ var flattenStyle = require('flattenStyle');
  *  - It also allows to send the style only once through the bridge. All
  * subsequent uses are going to refer an id (not implemented yet).
  */
-class StyleSheet {
-  static flatten: typeof flattenStyle;
+module.exports = {
+  /**
+   * The following constants are implemented for use in StyleSheets:
+   *
+   * - `HAIRLINE_WIDTH`:
+   *    - This is defined as the width of a thin line on the platform. It
+   *      can be used as the thickness of a border or division between two
+   *      elements. Example:
+   *      ```
+   *        {
+   *          borderBottomColor: '#bbb',
+   *          borderBottomWidth: StyleSheet.Constants.HAIRLINE_WIDTH
+   *        }
+   *      ```
+   *
+   *      This constant will always be a round number of pixels (so a line
+   *      defined by it look crisp) and will try to match the standard
+   *      width of a thin line on the underlying platform. However, you should
+   *      not rely on it being a constant size, because on different platforms
+   *      and screen densities its value may be calculated differently.
+   */
+  Constants: {
```

Comment:
Can you remove this extra level of indirection. We try to avoid unnecessary nesting as much as possible.

```
StyleSheet.hairlineWidth
```